src/test/kotlin/no/nav/syfo/cronjob/behandlendeenhet/PersonBehandlendeEnhetCronjobSpek.kt

```diff
@@ -161,14 +159,9 @@ object PersonBehandlendeEnhetCronjobSpek : Spek({
 
                         database.connection.dropData()
 
-                        database.connection.use {
-                            personoppgavehendelseService.processPersonoppgavehendelse(
-                                connection = it,
-                                kPersonoppgavehendelse = oversikthendelse,
-                                callId = UUID.randomUUID().toString(),
-                            )
-                            it.commit()
-                        }
+                        personoversiktStatusService.createOrUpdatePersonoversiktStatuser(
+                            personoppgavehendelser = listOf(oversikthendelse)
+                        )
```

Comment:
Her hadde vi egentlig brukt consumer-kode for å generere testdata til databasen. Nå bruker vi servicen, men kan jo hende det hadde vært like greit å lage en test-db metode som tok inn en hendelse og lagde en personoversiktstatus basert på det bare? Da frikobler man testene fra logikk som handler om konsumering eller servicen 🤷🏼‍♂️